k-distribution/tests/regression-new/map-symbolic-tests-haskell/lookup-23-spec.k.out

```diff
@@ -1 +1,16 @@
-#Top
+  #Not ( {
+    X
+  #Equals
+    y
+  } )
+#And
+  <k>
+    lookupResult ( ( X:MyId |-> 3 ) [ x <- 5 ] [ X:MyId ] ~> . ) ~> .
+  </k>
+#And
+  {
+    true
+  #Equals
+    X:MyId in_keys ( ( X:MyId |-> 3 ) [ x <- 5 ]
```

Comment:
I think the result above is correct, but this should be simplified further. The `X` is obviously in the keys of the map.